src/azul/plugins/repository/dss/__init__.py

```diff
@@ -130,6 +132,11 @@ def fetch_bundle(self, source: str, bundle_fqid: BundleFQID) -> Bundle:
                  time.time() - now, bundle.uuid, bundle.version)
         return bundle
 
+    @cache
+    def repository_source(self, source: str) -> Source:
+        self._assert_source(source)
+        return Source(name=source, uuid='filler_uuid_for_dss')
```

Comment:
Should we generate a UUID and hard code it here?

Reply:
I was thinking about doing this, but I want to get @hannes-ucsc opinion on what I should do here. Leaving it the way it is also draws attention to the fact that there's no validation on the UUID. I'm not sure if that should change.